config/capz/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller:latest
+      - image: localhost:5000/ci-e2e/cluster-api-azure-controller-amd64:20230518170459
```

Comment:
I think you may have mistakenly committed this after running an E2E test. The latest changes on main should have a fix to prevent this.

Reply:
Good catch! Sorry about that!